src/librustc_back/target/mod.rs

```diff
@@ -357,6 +358,7 @@ impl Target {
             i386_apple_ios,
             x86_64_apple_ios,
             aarch64_apple_ios,
+            aarch64_linux_android,
```

Comment:
Pure cosmetics: can this line be moved to linux group or may be extract all android targets in it is own group